docker-compose/compose.yaml

```diff
@@ -186,6 +226,12 @@ services:
         http://shared-file-store:8099/alfresco/api/-default-/private/sfs/versions/1/file/
       SPRING_ACTIVEMQ_BROKERURL: nio://activemq:61616
       SPRING_ELASTICSEARCH_REST_URIS: http://elasticsearch:9200
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:8080/actuator/health"]
+      interval: 20s
+      timeout: 20s
+      retries: 3
+      start_period: 60s
```

Comment:
does it really take so much time to start with es ready?

Reply:
Changed it to 20s